sdk/boards/sonata-prerelease.json

```diff
@@ -63,6 +63,10 @@
         "plic": {
             "start" : 0x88000000,
             "end"   : 0x88400000
+        },
+        "pwm": {
```

Comment:
@HU90m Is this also present in the 0.2 release?

Reply:
> @HU90m Is this also present in the 0.2 release?

I've just checked for @HU90m, and while the PWM existed in the code at the point of the 0.2 release, it wasn't hooked up to any pins in the RTL and so there's no way to use it. I think in this case it doesn't make sense to add this to the 0.2 board file also.